source/src/PascalCompiler/Grammar/Nonterminals.h

```diff
@@ -111,7 +112,67 @@ enum NonterminalSymbols {
     VarSection,                              // 197
     VarSection1,                             // 198
     VarSection2,                             // 199
-    FinalType,
+    AdditiveExpression,
+    AdditiveExpression1,
```

Comment:
As I see it, you changed some names in grammar but forgot to change them in this enum. Please rename them here too.